### PR TITLE
Issue #2301: [UX] Rename "Admin content" and "Comments" to "Manage co…

### PR DESCRIPTION
**Pull request by jenlampton**
Fixes https://github.com/backdrop/backdrop-issues/issues/2301
Replaces https://github.com/backdrop/backdrop/pull/1665

### CODE REVIEW

**Comment by Dinornis** [comment]
Navigating to this PR URL is initiating an install.

**Comment by jenlampton** [comment]
closing and opening for a new sandbox.

File: core/modules/views/tests/views_ui.test [hunk 1]
```diff
@@ -25,7 +25,7 @@ class ViewsUIWizardBasicTestCase extends ViewsUIWizardHelper {
   function testViewsWizardAndListing() {
     // Check if we can access the main views admin page.
     $this->backdropGet('admin/structure/views');
-    $this->assertLinkByHref(url('admin/config/development/configuration/single/export/Views/views.view.node_admin_content'));
+    $this->assertLinkByHref(url('admin/config/development/configuration/single/export/Views/views.view.manage_content'));
```
Comment (quicksketch) [inline core/modules/views/tests/views_ui.test]: This change still needs to be reverted, as the view name has not changed now.

**Comment by quicksketch** [comment]
Merged into 1.x for 1.6.0.

**Pull request merged**